arviz/data/inference_data.py

```diff
@@ -91,7 +91,15 @@ def to_netcdf(self, filename, compress=True):
         return filename
 
     def __add__(self, other):
-        """Concatenate two InferenceData objects."""
+        """Concatenate two InferenceData objects.
+
+        example
```

Comment:
Capitalization is Incorrect

Standard is `Examples` 

https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html